client/cmd/internal/logger/printer.go

```diff
@@ -6,15 +6,16 @@ import (
 	pb "github.com/odpf/optimus/protos/odpf/optimus/core/v1beta1"
 )
 
-func PrintLogStatus(logger log.Logger, logStatus *pb.Log) {
-	switch logStatus.GetLevel() {
-	case pb.Level_LEVEL_INFO:
+func PrintLogStatus(logger log.Logger, logStatus *pb.Log, verbose bool) {
```

Comment:
Let's avoid using flag arguments in the function as they are one of smell codes. Instead of sending verbosity status into this function, what do you think if we split the function itself into two? One of them is for printing with verbose (for example `PrintLogStatusVerbose`) and another without verbose (such as `PrintLogStatus`). Think of it like `fmt.Print` and `fmt.Println`.

Reply:
Got it, so control flow will be handled in the outside PrintLog func